vowpalwabbit/distributionally_robust.cc

```diff
@@ -7,6 +7,44 @@
 
 namespace VW
 {
+namespace model_utils
+{
+size_t process_model_field(
+    io_buf& io, VW::distributionally_robust::Duals& duals, bool read, const std::string&, bool text)
+{
+  if (io.num_files() == 0) { return 0; }
+  process_model_field(io, duals.unbounded, read, "_duals_unbounded", text);
+  process_model_field(io, duals.kappa, read, "_duals_kappa", text);
+  process_model_field(io, duals.gamma, read, "_duals_gamma", text);
+  process_model_field(io, duals.beta, read, "_duals_beta", text);
+  process_model_field(io, duals.n, read, "_duals_n", text);
+  return 0;
```

Comment:
This should return the total bytes read or written. Adding up the other calls should suffice.